assets/fonts/fonts.css

```diff
@@ -0,0 +1,47 @@
+/**
+ * Source Sans Pro
+ *
+ * Use for: labor
+ * Font-family: 'Source Sans Pro', sans-serif
+ * Available styles:
+ * - font-style: normal / font-weight: normal
+ * - font-style: italic / font-weight: normal
+ * - font-style: normal / font-weight: bold
+ * - font-style: italic / font-weight: bold
+ */
+@font-face {
+    font-family: 'Source Sans Pro';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Source Sans Pro'),
+         local('SourceSansPro-Regular'),
+         url('./sourcesanspro-regular.woff2') format('woff2'),
+         url('./sourcesanspro-regular.woff') format('woff');
```

Comment:
I don't know, I picked it based on the font name. Totally open to remove it. @m4dz ?